scripts/addingAdminKeystoLC.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#adding keys to existing users via LC
+curl ${URL_KEYS[@]} --output /tmp/keys
```

Comment:
Using a known temp location can be a security risk. Use mkstmp to create a unique file location.